about.html

```diff
@@ -0,0 +1,91 @@
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="./about.html">About</a></li>
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="">Join</a></li>
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="">Sponsor</a></li>
+            <li class="nav-item"><a class="text-decoration-none text-dark fw-normal" href="">News</a></li>
+            <li class="nav-item special p-2 px-4"><a class="text-decoration-none text-dark fw-normal" href="">Our Offers</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main class="about-us bg-light">
+        <section class="hero d-flex flex-column align-content-center align-items-center ">
+            <h1 class="title-about text-center ">Creative Cars Renting Website 2022</h1>
+            <hr class="bord"></hr>
+            <p class="home-para text-center">
+                That wonderful feeling – you start the engine and your adventure begins…
+                At Carento.com everything we do is about giving you the freedom to discover more. We’ll move mountains to find you the right rental car, and bring you a smooth, hassle-free experience from start to finish. Here you can find out more about how we work.
+            </p>
+          </section>
+          <hr class="seperator">
+          <section class="begin d-flex flex-column justify-content-center align-content-center align-items-center">
+            <h3 class="section-title">
+            See our begining
+            </h3>
+            <hr class="bord">
+            <p class="para">This is the first two cars which we begin with in our journey</p>
+            <div class="begin-imgs d-flex flex-column">
+                <div class="pic pic1">
+                    <p class="car text-center lh-5"><span class="year">1989</span><br/>This is Our First Car</p>
+                </div>
+                <div class="pic pic2">
+                    <p class="car text-center lh-5"><span class="year">1990</span><br/>This is Our Second Car</p>
+                </div>
+            </div>
+          </section>
+          <section class="partners d-flex flex-column align-items-center justify-content-start mt-4">
+            <h3 class="section-title">Partnes</h3>
+            <hr class="bord">
+            <div class="partners-logo d-flex flex-row align-items-center justify-content-start mt-4">
+              <img src="./assets/partners/careem.png" alt="careem" class="partner">
+              <img src="./assets/partners/uber.png" alt="uber" class="partner">
+              <img src="./assets/partners/behance.png" alt="behance" class="partner">
+            </div>
+          </section>
+      </main>
+      <footer class="d-flex flex-row align-items-center">
+        <h2 class="carento">Carento</h2>
+        <p class="credit">2022 Renting Cars Website.<br/>Some Rights Reversed.</p>
+      </footer>
```

Comment:
- [ ] This should be a black footer without partners according to the design. Please fix this.
### Your implementation
![image](https://user-images.githubusercontent.com/45460192/190658295-6962e62f-bf2c-445d-b0eb-84442f36c0f3.png)

### Required
![image](https://user-images.githubusercontent.com/45460192/190658135-cb6ae82a-f164-4a66-af18-0cf9a7aaa3b3.png)